### PR TITLE
About page

### PR DESCRIPTION
**Pull request by mamadOuologuem**

### CODE REVIEW

File: src/components/about/WhatMakesUsUnique.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  { title: 'Open-source', subTitle: 'We foster collaboration', image: image2 },
+  { title: 'Micro-volunteering', subTitle: 'we accelerate change', image: image3 },
+];
+
+const useStyles = makeStyles({
+  cardsContainer: {
+    display: 'flex',
+    position: 'relative',
+    width: 'calc(100vw - 8px)',
+    left: 'calc(-1 * (100vw - 100%) / 2)', // to centered the div
```
Comment (bencpeters) [inline src/components/about/WhatMakesUsUnique.js]: Does this work responsively? Will it get weird if the page gets resized?

File: src/components/about/WhoWeAreBlock.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    as a software engineer in the Silicon Valley. My desire to code for good led me to work for a tech nonprofit,
+    Bayes Impact. \n\nWitnessing the huge need for technology in the social impact sector, I decided to co-create Ovio,
+    a tech volunteering platform. I am passionate about climate change and human rights. Since graduate school,
+    I have engaged my time and skills to support these causes.`,
+    photo: photoEric,
+  },
+  {
+    name: 'Ben Peters',
+    position: 'Director of Engineering',
+    linkedInUrl: 'https://www.linkedin.com/company/oviohub/',
```
Comment (bencpeters) [inline src/components/about/WhoWeAreBlock.js]: https://www.linkedin.com/in/ben-peters-19463329/

File: src/components/Footer.js [hunk 3]
```diff
@@ -1,48 +1,79 @@
...
-  const baseClasses = useStylesBase();
+  const { muiGridBlockContainer, muiIconLogo } = useStylesBase();
+  const { lightIcon, itemContainer, itemText, links } = useStyles();
   return (
-    <Grid container justify="flex-end">
-      <Grid item container sm={6} direction="column">
-        <Icon className={baseClasses.muiIconLogo} component="img" src={logo} alt="Ovio" />
+    <Grid className={muiGridBlockContainer} container justify="flex-end">
+      <Grid item container xs={6} direction="column">
+        <img className={muiIconLogo} component="img" src={logo} alt="Ovio" />
```
[the remark this answers is not in the document]
Comment (mamadOuologuem) [reply src/components/Footer.js]: Mistake :)

File: src/components/about/WhatMakesUsUnique.js (hunk 1 above)
Comment (mamadOuologuem) [reply src/components/about/WhatMakesUsUnique.js]: Yes that is what I was talking about with Eric a few days before.
 I changed it :smile:

File: src/components/Footer.js [hunk 4]
```diff
@@ -44,33 +44,33 @@ const Footer = () => {
   return (
     <Grid className={muiGridBlockContainer} container justify="flex-end">
       <Grid item container xs={6} direction="column">
-        <img className={muiIconLogo} component="img" src={logo} alt="Ovio" />
+        <img className={muiIconLogo} src={logo} alt="Ovio" />
         <Grid>
-          {socialMedias.map(item => (
-            <a key={item.link} href={item.link} target="_blank" rel="noreferrer noopener">
-              <Icon component="img" className={lightIcon} src={item.image.src} alt={item.image.alt} />
+          {socialMedias.map(({ link, image: { imgSrc, imgAlt } }) => (
```
Comment (bencpeters) [inline src/components/Footer.js]: No need to change this, but it's totally find to leave these as `{src, alt}` and then use `src={src}` and `alt={alt}` on the components.

**Pull request merged**